From e77b595b4ba92cd0cfa947aeb1c8e1f8aee5e713 Mon Sep 17 00:00:00 2001
From: Kristian Amlie <kristian.amlie@northern.tech>
Date: Tue, 3 Dec 2019 13:29:57 +0100
Subject: [PATCH 1/1] Revert "efi_loader: disable EFI_LOADER on
 vexpress_ca15_tc2 and ca9x4"

We use EFI loading in Mender.

This reverts commit f95b8a4b5f64fd43e88c756662e77330dce8c007.
---
 configs/vexpress_ca15_tc2_defconfig | 1 -
 configs/vexpress_ca9x4_defconfig    | 1 -
 2 files changed, 2 deletions(-)

diff --git a/configs/vexpress_ca15_tc2_defconfig b/configs/vexpress_ca15_tc2_defconfig
index 904c756..3386928 100644
--- a/configs/vexpress_ca15_tc2_defconfig
+++ b/configs/vexpress_ca15_tc2_defconfig
@@ -32,4 +32,3 @@ CONFIG_SMC911X_32_BIT=y
 CONFIG_BAUDRATE=38400
 CONFIG_CONS_INDEX=0
 CONFIG_OF_LIBFDT=y
-# CONFIG_EFI_LOADER is not set
diff --git a/configs/vexpress_ca9x4_defconfig b/configs/vexpress_ca9x4_defconfig
index 06fcfee..65da6e22 100644
--- a/configs/vexpress_ca9x4_defconfig
+++ b/configs/vexpress_ca9x4_defconfig
@@ -31,4 +31,3 @@ CONFIG_SMC911X_32_BIT=y
 CONFIG_BAUDRATE=38400
 CONFIG_CONS_INDEX=0
 CONFIG_OF_LIBFDT=y
-# CONFIG_EFI_LOADER is not set
-- 
2.7.4

